src/js/components/organisms-page/resellers/resellers.js

```diff
@@ -1,3 +1,14 @@
+import {TweenLite} from 'gsap';
+import {Power3} from 'gsap';
+import Defaults from 'constants/defaults';
+
+const resellerCardTransitionDuration = Defaults.defaultDuration,
+    resellerCardOffset = Defaults.buildingUnit * 5,
+    cardsYPos = [],
+    delay = 100;
```

Comment:
You might want to use 0.1 here so you don't have to use in the setDelay function